### ISSUE TITLE
Request to post a demo on the website

### ISSUE THREAD

**Issue by lhg99** [gradio-app/world-of-demos#12]
Hello,
We are students at Dankook University in Korea.
The goal of the lecture we are taking is to contribute to open source SW. so we chose gradio open source library.
We wanted to contribute by creating a demo.
We sent the request to the radio-app/gradio in repo. And I got an answer that it would be better to send a request here.

It's a demo that we made.
https://github.com/lhg99/Gradio-demo
We attach our Githeub link, so please read the Readme.md file.

If this demo is suitable for posting websites, we want it to be posted in the Tabular Data & Plots category.
Please review it thoroughly and reply to me a reply.

Thank you!

**Comment by abidlabs** [comment]
Nice! Yes, is the demo hosted anywhere? I'd recommend hosting it on Spaces (https://huggingface.co/spaces), and then you can open a PR to link it from the Tabular Data & Plots category

**Comment by lhg99** [comment]
We hosted our demo on Spaces! URL is https://huggingface.co/spaces/lhg99/gradio-demo

This demo shows 2022 year weather in table and graph by selecting the month, day, weather elements, and location.
We think this demo is good example for Tabular Data & Plots category.

Please review it thoroughly and reply to me a reply.

**Comment by abidlabs** [comment]
Thanks @lhg99 looks good to me. Can you please create  PR to add this to the right section?

**Comment by lhg99** [comment]
It's our first time contributing open source, so we don't know where to add PR.

We forked awesome-demos.
We will edit README files and will add a pull request.
Is this method right? If it's right, could you add Tabular Data & Plots catefory in README files?
If not, could you tell us the right section where we add a PR?

Thank you!

**Comment by abidlabs** [comment]
That's correct @lhg99. Actually instead of creating a new section, what about adding it to the table under the Science category?

**Comment by lhg99** [comment]
We sent a pull request. @abidlabs 

Can we know how the demos in the demos category on the Gradio website are posted?

We attach the examples of demos on gradio website.
![image](https://github.com/gradio-app/awesome-demos/assets/94951045/dca5c5ee-1d73-4968-9226-4c23dfee0b61)

**Comment by lhg99** [comment]
Thank you for accepting my PR!
Demo is running on huggingface, but demo status in README.md is down.
Could you modify my demo status part?

Thank you!

**Comment by abidlabs** [comment]
All good now it seems!
